src/lib/webpack/run-webpack.js

```diff
@@ -136,7 +136,7 @@ export function writeJsonStats(stats) {
 	function strip(stats) {
 		stats.modules.forEach(stripLoaderFromModuleNames);
 		stats.chunks.forEach(c => {
-			(c.mapModules!=null ? c.mapModules(Object) : c.getModules()).forEach(stripLoaderFromModuleNames);
+			(c.modules || (c.mapModules!=null ? c.mapModules(Object) : c.getModules())).forEach(stripLoaderFromModuleNames);
```

Comment:
`getModules()` is deprecated right? 

That's reason why we see that weird message when we run `preact build`